NitroxServer/GameLogic/Players/PersistedPlayerData.cs

```diff
@@ -94,7 +96,7 @@ public static PersistedPlayerData FromPlayer(Player player)
             NitroxId = player.GameObjectId,
             IsPermaDeath = player.IsPermaDeath,
             CompletedGoals = new(player.CompletedGoals),
-            PingInstancePreferences = new(player.PingInstancePreferences)
+            PlayerPreferences = new(player.PingInstancePreferences.ToDictionary(m => m.Key, m => m.Value), player.PinPreferences.ToList())
```

Comment:
Any reason why we convert to dictionary for persistence?  usually we keep persisted data in flat list format.

Reply:
[PingInstancePreference](https://github.com/SubnauticaNitrox/Nitrox/blob/master/NitroxModel/DataStructures/GameLogic/PingInstancePreference.cs)s only contain the very least data (color and visibility) and the dictionary is the only place where the PingInstancePreferences' keys are held. So if we really want the ping preferences to be persisted as a list, there'll just be onre more (but unused) field in the data object without much else being affected.
To me, a dictionary is the simplest and lightest way of achieving the persistence/using in this situation but please tell me your preference on this.